crates/db_schema/src/impls/community.rs

```diff
@@ -311,9 +311,9 @@ impl ApubActor for Community {
     protocol_domain: &str,
   ) -> Result<Community, Error> {
     let conn = &mut get_conn(pool).await?;
+    let from_actor_id = format!("{}/c/{}", protocol_domain, community_name).to_lowercase();
     community
-      .filter(lower(name).eq(lower(community_name)))
-      .filter(actor_id.like(format!("{protocol_domain}%")))
```

Comment:
These wildcard queries are always unavoidably slow. To fix, I build the `actor_id`, and use the existing `lower(actor_id)` index.

Reply:
This is also used to read federated actors from other platforms. These have different URL formats so its not going to work. 

Instead you can check the domain via `instance` table. Checking the protocol seems unnecessary.